airflow/cli/commands/db_command.py

```diff
@@ -49,15 +49,15 @@ def resetdb(args):
 def upgradedb(args):
     """Upgrades the metadata database"""
     print("DB: " + repr(settings.engine.url))
-    if args.revision and args.version:
-        raise SystemExit("Cannot supply both `--revision` and `--version`.")
+    if args.to_revision and args.to_version:
+        raise SystemExit("Cannot supply both `--to-revision` and `--to-version`.")
     if args.from_version and args.from_revision:
         raise SystemExit("Cannot supply both `--from-revision` and `--from-version`")
     if (args.from_revision or args.from_version) and not args.show_sql_only:
         raise SystemExit(
             "Args `--from-revision` and `--from-version` may only be used with `--show-sql-only`"
```

Comment:
Looks like when we use `--from-version`  or `--from-revision` we must use `--show-sql-only`. Should we make it to run real migration if the flag is used and raise error only if the user's version/revision is not the same with the supplied `--from-revision` /`--from-version`

Reply:
That's a good idea, but it's outside the scope of this PR.